src/test/java/com/google/javascript/clutz/DeclarationSyntaxTest.java

```diff
@@ -85,7 +85,8 @@ public boolean accept(File dir, String name) {
           "--lib",
           "es5,dom,es2015.iterable",
           "--noImplicitAny",
-          "--strictNullChecks");
+          "--strictNullChecks",
+          "--noStrictGenericChecks");
```

Comment:
add a comment next to this flag, similar to the one in the cl description. Long-term we should attempt removing this.

Reply:
Done.